app/models/order_item.rb

```diff
@@ -1,4 +1,7 @@
 class OrderItem < ApplicationRecord
   belongs_to :order
   belongs_to :product
+  def self.ransackable_attributes(auth_object = nil)

```

Comment:
Why you did it like that?

Reply:
I already informed you on slack 
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/5a496e5c-644f-4d25-b802-9eac39465f3a">